server/proxy/proxy.go

```diff
@@ -156,7 +156,11 @@ func (pxy *BaseProxy) startListenHandler(p Proxy, handler func(Proxy, net.Conn,
 				// if listener is closed, err returned
 				c, err := l.Accept()
 				if err != nil {
-					xl.Info("listener is closed")
+					xl.Warn("listener is closed: %s", err)
```

Comment:
https://github.com/fatedier/golib/blob/master/net/mux/mux.go#L134
Can you just copy that codes here with additional log?

Reply:
This log should be moved to line 178